x/http_secure_redirect.go

```diff
@@ -145,10 +143,8 @@ func SecureRedirectTo(r *http.Request, defaultReturnTo *url.URL, opts ...SecureR
 
 	return nil, errors.WithStack(herodot.ErrBadRequest.
 		WithID(text.ErrIDRedirectURLNotAllowed).
-		WithReasonf("Requested return_to URL %q is not allowed.", returnTo).
-		WithDebugf("Allowed domains are: %v", strings.Join(lo.Map(o.allowlist, func(u url.URL, _ int) string {
-			return u.String()
-		}), ", ")))
+		WithReasonf("Requested return_to URL %q is not allowed.", returnTo),
```

Comment:
Log this instead then? Or trace?

Reply:
I don't think that adds any value, because you can always check the project configuration.

I added this a while back mainly to make live for a developer implementing Ory a little easier, but in this configuration we're leaking this information to end users if they deliberately trigger this error (e.g. by providing an illegal `return_to`).

Reply:
ok